robyn/openapi.py

```diff
@@ -138,11 +139,15 @@ class OpenAPI:
 
     info: OpenAPIInfo = field(default_factory=OpenAPIInfo)
     openapi_spec: dict = field(init=False)
+    is_pre_configured: bool = False
```

Comment:
we should name it something else. `is_pre_configured` is a wrong term. should name it in the direction of override.